src/main/java/com/nike/cerberus/operation/composite/CreateEnvironmentOperation.java

```diff
@@ -87,8 +79,11 @@
             // Upload to S3 for CMS to download at service start
             new ChainableCommand(new CreateCmsConfigCommand()),
 
+            // Generate the instance profile to use when creating the CMS Cluster
+            new ChainableCommand(new CreateInstanceProfileCommand()),
+
             // Create the CMS Cluster Stack
-            new ChainableCommand(new CreateCmsClusterCommand()),
+            new ChainableCommand(new CreateCmsAsgCommand()),
```

Comment:
Is there a reason you call the two separate commands instead of the composite command you created?

Reply:
I wanted to continue following the pattern of calling basic commands from a composite command instead of calling a composite command from a composite command.

Reply:
I also feel like it's a little bit easier to debug when the functionality isn't abstracted by the use of a composite command.